javascript/javascript2/week1/animal-name-generator/index.js

```diff
@@ -0,0 +1,45 @@
+//Spirit animal part
+
+//const userName = prompt("Enter your name");
+
+const spiritAnimalNames = [
+    'The crying butterfly',
+    'The full moon wolf',
+    'The dangerous tiger',
+    'The gigantic giraffe',
+    'The big old elephant',
+];
+
+//getting random numbers from spiritAnimalNames
+let randomNumber = Math.floor(Math.random() * spiritAnimalNames.length);
+
+const myNewButton = document.querySelector('#myButton');
+
+function clickFunction() {
+    myNewButton.addEventListener('click', function () {
+        if (document.querySelector('#click').checked) {
+            getUsernameWithSpirit();
+        }
+    });
+}
+
+function hoverFunction() {
+    myNewButton.addEventListener('mouseover', function () {
+        if (document.querySelector('#hover').checked) {
+            getUsernameWithSpirit();
+        }
+    });
+}
+
```

Comment:
addEventListener will work when you perform the actions, thats when you click, hover, etc. you don't need to enclose addEventListener inside these functions